sklearn/feature_selection/_sequential.py

```diff
@@ -36,6 +36,11 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin,
         to select. If float between 0 and 1, it is the fraction of features to
         select.
 
+    aborted_rate : float, default=None
+	If the score is not increasing above ths rate, stop adding new features
+	even if n_features_to_select has not been reached. If None (default)
+	all n_features_to_select will be selected.
```

Comment:
Maybe make it more explicit that this a threshold on the increment:

If the score is not incremented by at least `tol` between two consecutive feature addition or removal, stop adding or removing ...

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20145/commits/44300288cc81e213203febe6eddaf928fcbb48f1

Maybe I don't fully understand the code, but can the features be reduced?
Does that mean reducing from the original features?

Reply:
Yes, look at the docstring for `direction="backward"`.

Reply:
thanks, I understand !